lib/_http_client.js

```diff
@@ -129,6 +129,12 @@ function ClientRequest(options, cb) {
                       self._renderHeaders());
   }
 
+  this._ended = false;
+  this.res = null;
+  this.aborted = 0;
+  this.timeoutCb = null;
+  this.upgradeOrConnect = false;
```

Comment:
It's usually helpful to add a little comment on these, as it helps in the long run (see `_readableState` as an example).